dds_web/utils.py

```diff
@@ -16,17 +16,25 @@
 from contextlib import contextmanager
 import flask
 import sqlalchemy
+import pytz
 
 # Own modules
 from dds_web.database import models
 from dds_web import db, C_TZ
+import dds_web.utils
```

Comment:
This would import this file inside this file?

Reply:
Removing.